### ISSUE TITLE
TinyMc Editor issues

### ISSUE THREAD

**Issue by SpiritWolfie** [SocialEngine/phpv4-issues#669]
4.9 

managing announcements the TinyMC editor does not allow proper use of of the editor features (like bullet etc) this update also seems to take away the ability to space posts

Test

Test 2

instead it does
Test
Test

![image](https://cloud.githubusercontent.com/assets/9712738/25058541/6ed41398-212f-11e7-9d15-da35325f1be4.png)

When using the bullet function its only suppose to 'bullet' the highlighted text like editors are suppose to do, instead, it puts the bullet at the beginning, Sadly this is the case with the quote button and any other button as well..

in the above pic it shows I spaced the comments
but when you publish it

![image](https://cloud.githubusercontent.com/assets/9712738/25058553/b0951098-212f-11e7-82b4-b625a08000a4.png)

this is the results of publishing it.

**Comment by poeticjustice** [comment]
arrrrrrr man thats not good, i use announcements all the time :(

**Comment by DonnaScriptTechs** [comment]
I'll go try this on my demo site.

**Comment by DonnaScriptTechs** [comment]
Regarding spacing posts, I can confirm that. 

Regarding bullets, I can confirm that. I can also confirm that numbers are cut off for numbered lists. 

<img width="525" alt="tinymce" src="https://cloud.githubusercontent.com/assets/6908282/25062547/15be063e-219c-11e7-9c05-86d7edd696cd.png">

This is just in announcements as **it works fine in other areas**. 
<img width="468" alt="tinyblogfine" src="https://cloud.githubusercontent.com/assets/6908282/25062490/f7a34cdc-219a-11e7-8def-2cc468b376a7.png">

**Comment by Elshara** [comment]
Too bad announcements couldn't be linked with broadcast message
feature. That would be nice.
Maybe try the view page source area and then just fix up the html code
prior to publishing it.

**Comment by SpiritWolfie** [comment]
Also just a thought but TinyMC editor has been terrible for a long time, why not switch to something different? most platforms these days have moved away from tinymc because of all the issues it has.

**Comment by Elshara** [comment]
I miss pure html days.

**Comment by SpiritWolfie** [comment]
This was set to be fixed in 4.9.1, We finally managed to get 4.9.1 installed and this has not been fixed

Announcements still disregard any formatting, or spacing, I even made it a point to remove my post and and re-create it, and no go.....

**Comment by DonnaScriptTechs** [comment]
This wasn't fixed. We add a "fixed" label and close any that are fixed. We tried to get it for 4.9.1 but weren't able to.

**Comment by SpiritWolfie** [comment]
@DonnaScriptTechs :'(

Could we get rid of tinymc then please or get it switched out for something better.. like seriously (I just checked again and like no one uses tinymc anymore these days) just a suggestion.....

**Comment by DonnaScriptTechs** [comment]
Many like tinymce. Wordpress uses it, which is a hugely popular platform. I prefer a built in option with just some features instead of any external editor but that's just me. I think there is a feature request here to replace TinyMCE? Though the options that I see out there are not rating as high as Tiny so there might not be a better option right now.

**Comment by SpiritWolfie** [comment]
Just some thoughts about this with some other stuff

For TinyMC why not take a look at what Druple uses for text editor? theirs works flawlessly no matter how desperately hard I try to break it, I know many platforms have since moved away from TinyMC , While some still use it, that number is far and few between from my personal testing and research. I'm not sure what the other platforms are switching too to be honest, it might be a none open source editor and is a paid one., or one they themselves wrote.

For Flash, I would recommend using a HTML 5 uploader for both the admin panel (package manager), Photo uploads etc.. 
This would resolve so many problems by itself specifically with 'older machines' or browsers that hard block flash etc.. Many platforms have since moved away from flash due to the countless security issues that are found in flash, (I get allot of complaints from users unable to upload photos on my website while most can, but those who can not, Can upload fine anywhere else on the net or application.)

Change the way the HTACCESS works.. as it stands it uses the old way and a big 'security risk' of FollowSymlink - Please see 
https://www.google.com/search?q=Followsyntex+dangers&oq=Followsyntex+dangers&aqs=chrome..69i57.5544j0j7&sourceid=chrome&ie=UTF-8#q=Danger+of+FollowSymlinks

Multiple OS's installed by scratch out of the box (No configurations) has this blocked by default... from what we've seen, I would highly advise changing the way htaccess handles FollowSymlink. and use the new modern way of handling it as seen in many other platforms which got away from it like 5 to 6years ago <.<.....

Another Issue I'd like to see addressed is the hard coded addresses in SE, instead of having a line of code that calls for that info, their are endless amounts of files  with the addresses in it, Which makes it difficult to easily move SE from one server configuration to another, My server tech gave me a live demonstration this morning in regards to just how many files have the addresses in it, and I sat and watched the list go down for over 5+ Mins...

What druple does in this regard is they have a code in their file that other things can put a call in for to find out the locations of the stuff.. SE does not do this, which makes it much more complicated to move it without having to jump though hoops and loops , Without having to 'hire' a 'se expert' to go in and change all the hard coded addresses etc.....

Just some thoughts, I know its a bit off topic and I do apologize. (it sorta is relevant since we are discussing replacements etc.. and what not)

**Comment by Elshara** [comment]
CKEditor have you tried it?

**Comment by DonnaScriptTechs** [comment]
This is fixed for 4.9.2 when it is released. If you still have this issue after we release, please open a new bug report.
